### ISSUE TITLE
Add Lexend step 2 : VF

### ISSUE THREAD

**Issue by RosaWagner** [google/fonts#3203]
upstream: https://github.com/FontBureau/fb-lexend

- [x] Ask @davelab6 if we register LXND Axis (multi-axes, I don't even know if we can technically)
- [ ] Find out how to build a STAT table format 4 with gftools builder (https://github.com/googlefonts/gftools/issues/333)
- [x] Ask registration for SPAC axis (#3156)
- [x] Add Expanded ExtraBold
Add Loose Expanded ExtraBold (not needed for this VF)
- [x] Build system for wght, wdth, spac
- [x] Push back to FB
- [x] Set up static exports with LXND name (deca etc.)
- [ ] Push back to TJ (waiting for more info about LXND axis + how arabic is made)
- [ ] Package

-- BENCHED until SPAC axis is registered, and until we can build STAT table format 4 with the builder.

**Comment by RosaWagner** [comment]
This is on hold until SPAC axis is registered.

I don't think we need to register the LXND axis in the axis registry. I would be in favour of: 
- updating the Lexend static families with last fixes + arabic set
- integrating LXND only in the STAT table as multi-axes (format 4) with fallbacks deca, giga, mega etc. (when it becomes possible)

Indeed, having the deca,mega,etc instances in the ZIP file would create a conflict with the existing families (which has the medium weight as default regular) if users install statics from Lexend and Lexend Deca for example.

**Comment by RosaWagner** [comment]
Hyper Expansion (HEXP)
Expansion of inner and outer space of letterforms.
min:-24, default:0, max:24

- Fallbacks could follow the unit prefix (as it is in the Lexend families) and value could be power factor (eg. deca is 10^1 > 1, Mega is 10^6 > 6)
- Lexend would have min/default: 1, max=8
- Note: the actual Lexend families doesn't seem to follow this progression. It goes in alphabetic order, instead of power order (I don't know why)

![Capture d’écran 2021-05-28 à 15 10 20](https://user-images.githubusercontent.com/12222436/119989046-09475900-bfc7-11eb-951b-b5e35de3ffa7.png)

**Comment by davelab6** [comment]
I propose the axis be a % so it's self referential

**Comment by chrissimpkins** [comment]
Added to Q3 2022 milestone.  We need to address https://github.com/googlefonts/axisregistry/issues/11 then we are ready to onboard this family.

**Comment by chrissimpkins** [comment]
Waiting for italics fixes from design team

**Comment by chrissimpkins** [comment]
Status?

**Comment by RosaWagner** [comment]
Spreadsheet: https://docs.google.com/spreadsheets/d/1PIw5-VcnDVfl6kTF5jzGnpAHzeZ3_1b240uwpuWHXfQ/edit?usp=sharing

## Noted issues:
- bad interpolation (see interpolation report with gftools qa)
- accents move between masters with same outlines: masters with same outlines but different spacing need to keep the same outlines, anchors position etc. 
- extrapolation of SPAC axis is changing the outlines dramatically and it shouldn't (may be related to precedent issue, or issue with Glyphs's extrapolation algorithm). 
- alternate shapes for bold glyphs are not working (need to be turned into alternate layers, or coded in `rvrn` feature).

## About the SPAC axis
- SPAC range is too small, and spacing in general is too tight for any subfamily > Deca.
- In the source files, all masters with SPAC=100 have been extrapolated. When the spacing is reviewed, these masters should be deleted.
- The spacing can be improved, therefore we expect some diff between the original version and the update. But it doesn't mean that the average spacing should change. We have to keep the same "progression" through the subfamilies as the original ones.

### How the values are calculated

The reference glyphs to compare masters and instances is `H wght=400, wdth=100`.

`Deca` on the `SPAC` axis is located at `0`. H's sidebearing is `98u`, therefore `98u` is mapped at 0. 
The value on teh SPAC axis is a relative percentage change from the family’s default spacing. 
Therefore the spacing should double between 0 and 100.
So if `H`'s sidebearing is `98` at `0`, then `H`'s sidebearing should be `98*2=196` at `100`. 
So an increase of `100%` is a increase of `98u`.

Zetta (largest subfamily) has a H's sidebearing of `182u`.
Therefore between Deca and Zetta there is an increase of `182-98=84u`. 
If an increase of 100% is an increase of 98u, then an increase of 84u is an increase of (`84*100/98`) `86%`.
Therefore Zetta is at `86` on the SPAC axis.

The files were delivered with a tighter spacing for the default style (`H`'s sidebearing=`85u` for the default, `170u` for the largest style).
If `Deca(98)=0`, then `LexendMin(85)=-13`. 
If `196u=100%`, then `170u=73%`. 

We notice that the range delivered does not cover `Zetta`. Therefore `LexendMax` masters has been added to the source files by extrapolation to be able to export all styles . So when the spacing is re-done, these masters should be deleted!!

**Comment by davelab6** [comment]
Per our meeting today I've moved this to Q2 so @octaviopardo has time to focus on another large project :)

**Comment by chrissimpkins** [comment]
@octaviopardo something that is still planned in Q2?

**Comment by octaviopardo** [comment]
Yes! Already looking at it

**Comment by vv-monsalve** [comment]
After today's meeting, it was agreed to publish the font with `SPAC` and `wdth`, leaving the spacing from distributed fonts in the positive range and the current spacing in the negative range.

**Comment by vv-monsalve** [comment]
Job on this project has been resumed with an updated goal that will include both the original instances + Octavio's work on spacing.

**Comment by chrissimpkins** [comment]
Moving the onboarding of these updates back to Q3

**Comment by vv-monsalve** [comment]
After a new revision to the font, it was decided to use [SPAC](https://github.com/googlefonts/axisregistry/blob/main/Lib/axisregistry/data/spacing.textproto) + [XTRA](https://github.com/googlefonts/axisregistry/blob/main/Lib/axisregistry/data/x_transparent.textproto). The latter would be the best fit since only the horizontal transparent space is changing without the stroke being modified.

**Comment by vv-monsalve** [comment]
The design team is working on the glyphs improvements + spacing revision.

**Comment by vv-monsalve** [comment]
Current development needs the creator's opinion.

**Comment by chrissimpkins** [comment]
@octaviopardo Emma will now be working with you to wrap up the work on this project @emmamarichal

**Comment by latin-1** [comment]
Why has [the repository](https://github.com/googlefonts/lexend) been archived? Does this mean that Lexend will not receive any further updates?

**Comment by emmamarichal** [comment]
@latin-1 still in progress, I guess the repo have been automatically archived, we will re-open it later.

**Comment by chrissimpkins** [comment]
> Why has [the repository](https://github.com/googlefonts/lexend) been archived? Does this mean that Lexend will not receive any further updates?

It is not an intentional change in the project status. It looks like this may be a relatively new automated archive policy that our google org owners defined across repositories.  Thanks for flagging it.  I unarchived the repository.

**Comment by chrissimpkins** [comment]
The changes from Octavio are under review by Bonnie.  I am moving this project update to tentatively onboard in Q2.  The schedule will be driven by her review response.

**Comment by brzzdev** [comment]
I'm a little confused. There was an italics version done at some point, but they're blocked by this work, right?

There are two tickets around italics:
- https://github.com/ThomasJockin/readexpro/issues/21
- https://github.com/googlefonts/lexend/issues/14#issuecomment-1340572250

It was apparently coming Spring 2022, then Q3 2023

This ticket was Q2 or Q3 of this year, and now blocked.

I just wondered if we could get some updated timelines here. 
It's a fantastic font, and I look forward to having italics so I can use it in more places.

**Comment by arjpar** [comment]
> I'm a little confused. There was an italics version done at some point, but they're blocked by this work, right?
> 
> There are two tickets around italics:
> 
>     * [Bold and italic ThomasJockin/readexpro#21](https://github.com/ThomasJockin/readexpro/issues/21)
> 
>     * [Italic? googlefonts/lexend#14 (comment)](https://github.com/googlefonts/lexend/issues/14#issuecomment-1340572250)
> 
> 
> It was apparently coming Spring 2022, then Q3 2023
> 
> This ticket was Q2 or Q3 of this year, and now blocked.
> 
> I just wondered if we could get some updated timelines here. It's a fantastic font, and I look forward to having italics so I can use it in more places.

@emmamarichal Do you mind clearing up any confusion? I’m confused as well. Why is it blocked?

**Comment by emmamarichal** [comment]
The project is still in progress and some changes need to be reviewed and discussed by the team. Thank you for your patience!

**Comment by arjpar** [comment]
For those wondering, I created a repository that has Lexend with italics (slanted). You can download the repository with the fonts [here](https://github.com/arjpar/Lexend). I took Octavio Pardo’s Lexend variable font and froze the axes to create the slanted Lexend.

**Comment by petertwise** [comment]
@emmamarichal - Is there anything we can do to help test to move this forward?
this is marked "upstream is working on it"... but I don't see any changes to https://github.com/googlefonts/lexend/commits/wdth-spac-axes for 2 years.

**Comment by petertwise** [comment]
Our work around for now:

Download the official release woff2 to handle non-italics:
https://cdn.jsdelivr.net/fontsource/fonts/lexend:vf@latest/latin-wght-normal.woff2

Grab the development branch TTF to handle italics:
https://github.com/googlefonts/lexend/blob/wdth-spac-axes/fonts/lexend/variable/Lexend-Italic%5BSPAC%2Cwdth%2Cwght%5D.ttf

Put them together:

```scss
/* ---- Lexend Variable -- https://fontsource.org/fonts/lexend ---- */

/* normal */
@font-face {
	font-weight: 100 900;
	font-family: "Lexend";
	font-style: normal;
	src:
		local(""),
		url(../../fonts/lexend_latin-wght-normal.woff2) format("woff2 supports variations"),
		url(../../fonts/lexend_latin-wght-normal.woff2) format("woff2-variations");
	font-display: swap;
}

/* italic */
/*
This is a temporary fix until an official variable font is available.
See https://github.com/google/fonts/issues/3203
Beta italic variable font taken from here:
https://github.com/googlefonts/lexend/tree/wdth-spac-axes/fonts/lexend/variable
*/

@font-face {
	font-weight: 100 900;
	font-family: "Lexend";
	font-style: italic;
	src:
		local(""),
		url(../../fonts/lexend_latin-spac-wdth-wght-italic.ttf) format("truetype supports variations"),
		url(../../fonts/lexend_latin-spac-wdth-wght-italic.ttf) format("truetype");
	font-display: swap;
}
```

This is working fine for now - certainly at least better than just letting each browser decide how/if to implement fake slant.

**Comment by brzzdev** [comment]
Still due 2026 Q1? 👀

**Comment by jthrilly** [comment]
Seems like there is some community appetite to help get this over the line. For over a year now people have been asking how they can help and what this is blocked on. Maybe if the process was less opaque you could harness some of that energy, and we could take some work off of your plate?

**Comment by emmamarichal** [comment]
Hello! We are currently working on it. Thank you for your patience!

**Comment by apastuszak** [comment]
This entire repo was archived on the 16th of April. Is the Italic version dead?

**Comment by brzzdev** [comment]
Damn, I really hope not 😞

**Comment by Benjamin-Lee** [comment]
Eagerly waiting on this too! Would be happy to help if I can but also not sure what's the status.

**Comment by arjpar** [comment]
It is being actively worked on by octavio as of recent. I already took his recent work and created static italics for lexend. Even more recent than the lexend italics I have published on my github @Benjamin-Lee

**Comment by plazman30** [comment]
@arjpar What's the linik to Octavio's repo?

**Comment by brzzdev** [comment]
Looks like it's https://github.com/octaviopardo/lexend/tree/NewSuggested
